### PR TITLE
Fix int to string conversion and some message formatting directives

### PR DESCRIPTION
**Pull request by epapbak**
# Description

Small fixes

## Type of change

- Refactor (refactoring code, removing useless files)

## Testing steps

Compiling the code and running go test

## Checklist
* [x] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: PoC-differ/differ.go [hunk 1]
```diff
@@ -177,7 +177,7 @@ func processReportsByCluster(ruleContent map[string]types.RuleContent, impacts t
 			continue
 		}
 
-		notificationMsg := generateNotificationMessage(notificationConfig.ClusterDetailsURI, string(cluster.AccountNumber), string(cluster.ClusterName))
+		notificationMsg := generateNotificationMessage(notificationConfig.ClusterDetailsURI, fmt.Sprint(cluster.AccountNumber), fmt.Sprint(cluster.ClusterName))
```
Comment (tisnik) [inline PoC-differ/differ.go:180]: it is more time consuming operation - is it really needed there?

File: PoC-differ/differ.go [hunk 2]
```diff
@@ -390,11 +390,11 @@ func generateWeeklyNotificationMessage(advisorURI string, accountID string, dige
 	}
 
 	payload := types.EventPayload{
-		notificationPayloadTotalClusters:        string(digest.ClustersAffected),
-		notificationPayloadTotalRecommendations: string(digest.Recommendations),
-		notificationPayloadTotalIncidents:       string(digest.Incidents),
-		notificationPayloadTotalCritical:        string(digest.CriticalNotifications),
-		notificationPayloadTotalImportant:       string(digest.ImportantNotifications),
+		notificationPayloadTotalClusters:        fmt.Sprint(digest.ClustersAffected),
```
Comment (tisnik) [inline PoC-differ/differ.go:393]: dtto

File: PoC-differ/differ.go [hunk 3]
```diff
@@ -305,7 +305,7 @@ func processAllReportsFromCurrentWeek(ruleContent map[string]types.RuleContent,
 			Int("important notifications", digest.ImportantNotifications).
 			Msg("Producing weekly notification for ")
 
-		notification := generateWeeklyNotificationMessage(notificationConfig.InsightsAdvisorURL, string(account), digest)
+		notification := generateWeeklyNotificationMessage(notificationConfig.InsightsAdvisorURL, fmt.Sprint(account), digest)
```
Comment (tisnik) [inline PoC-differ/differ.go:308]: dtto

File: PoC-differ/differ.go (hunk 1 above)
Comment (epapbak) [reply PoC-differ/differ.go:180]: Not needed at all. I let sed do the job, and this happened ^^

File: PoC-differ/differ.go (hunk 2 above)
Comment (epapbak) [reply PoC-differ/differ.go:393]: In this case yes. All the values in the digest struct are integers.

File: PoC-differ/differ.go (hunk 1 above)
Comment (tisnik) [reply PoC-differ/differ.go:180]: :)

File: PoC-differ/differ.go (hunk 2 above)
Comment (tisnik) [reply PoC-differ/differ.go:393]: true, sorry

File: PoC-differ/differ.go (hunk 3 above)
Comment (epapbak) [reply PoC-differ/differ.go:308]: I'm thinking it's probably not needed, as in the final step I Marshall it all into a string. But if you don't mind, I'd keep this one like this until I add the generateWeeklyNotificationMessage UT, for "secure refactoring" =). WDYT?

Comment (tisnik) [reply PoC-differ/differ.go:308]: ok

**Review by tisnik — APPROVED** [review]
ok

**Pull request merged**